tests/constants.py

```diff
@@ -1,3 +1,4 @@
+uuid = "e8953ff6-be52-11ed-afa1-0242ac120002"
```

Comment:
You need to ensure, that this is a speaking constant name.

```suggestion
cu_recipient_endpoint_id= "e8953ff6-be52-11ed-afa1-0242ac120002"
```

Reply:
What do you mean exactly by 'speaking constant name'. I understood it as following: Instead of naming the constant "uuid", it should be named as "application_uuid" since "uuid" is already an argument for some functions.